book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
```

Comment:
- [ ] Let's kindly use getters and setters for the rental variable since it can  also be modify that's `attr_accessor :rental`